src/test/HostActivationTests/DotNetBuilder.cs

```diff
@@ -65,6 +65,67 @@ public DotNetBuilder AddMicrosoftNETCoreAppFramework(string version)
             return this;
         }
 
+        /// <summary>
+        /// Add a mock of the Microsoft.NETCore.App framework with the specified version
+        /// </summary>
+        /// <param name="version">Version to add</param>
+        /// <remarks>
+        /// Product runtime binaries are not added. All the added mock framework will contain is hostpolicy,
+        /// a mock version of coreclr, and a minimal Microsoft.NETCore.App.deps.json.
+        /// </remarks>
+        public DotNetBuilder AddMicrosoftNETCoreAppFrameworkMockCoreclr(string version)
```

Comment:
Coreclr -> CoreClr ?
(I'm going based on the casing of the `StatusCode::CoreClr*` error codes)

Reply:
Done